Evaluator/Social/signal_evaluator/signal.py

```diff
@@ -93,3 +96,35 @@ def _get_tentacle_registration_topic(self, all_symbols_by_crypto_currencies, tim
                 symbols += currency_symbols
         # by default no time frame registration for social evaluators
         return currencies, symbols, to_handle_time_frames
+
+
+class PumpAndDumpSignalEvaluator(evaluators.SocialEvaluator):
+    SERVICE_FEED_CLASS = Services_feeds.TelegramApiServiceFeed
+
+    MARKET_TO_TRADE = "BTC"
+    SUPPORTED_TELEGRAM_CHANNEL = {"WallStreetBets Events 🛸": r"The coin we have picked to pump today is :  #(.*)"}
+
+    async def _feed_callback(self, data):
+        is_from_channel = data[services_constants.CONFIG_IS_CHANNEL_MESSAGE]
+        if is_from_channel:
+            sender = data[services_constants.CONFIG_MESSAGE_SENDER]
+            if sender in self.SUPPORTED_TELEGRAM_CHANNEL.keys():
+                message = data[services_constants.CONFIG_MESSAGE_CONTENT]
+                signal = self._get_signal_message(self.SUPPORTED_TELEGRAM_CHANNEL[sender], message)
+                if signal is not None:
+                    self.eval_note = -1
+                    await self.evaluation_completed(signal,
+                                                    symbol_util.merge_currencies(signal, self.MARKET_TO_TRADE),
```

Comment:
I think we should add a method to detect the symbol from feed

Reply:
You mean to integrate the MARKET_TO_TRADE to SUPPORTED_TELEGRAM_CHANNEL ?